Paws4Me/Paws4Me/Controllers/Logic/SignInViewController.swift

```diff
@@ -0,0 +1,142 @@
+//
+//  SignInViewController.swift
+//  Paws4Me
+//
+//  Created by Kirsty-Lee Walker on 2022/02/26.
+//
+import UIKit
+
+protocol Validation {
+    func isValidCredentialsUsername(username: String) -> Bool
+    func isValidCredentialsPassword(password: String) -> Bool
+}
+
+let bottomLine = CALayer()
+let bottomLine2 = CALayer()
+
+class SignInViewController: UIViewController, Validation {
+    func isValidCredentialsUsername(username: String) -> Bool {
```

Comment:
Also slight naming enhancement here: 

`isValid(username: String) -> Bool`

Reply:
Noted, I renamed it to isValidCredentials